CONTRIBUTING.md

```diff
@@ -110,45 +110,42 @@ Or:
 
 For example:
 
-    npm test test/unit/template.html
+	npm test test/unit/template.html
 
 You can also run tests in the browser:
 
-    polyserve
+	polyserve
 
 Navigate to:
 
 [`http://localhost:8080/components/polymer/test/runner.html`](http://localhost:8080/components/polymer/test/runner.html)
 
 ### Running Polymer element unit tests
 
-To run the element unit tests, you need a global 
-install of `web-component-tester` or `polyserve` (or both). 
+To run the element unit tests, you need a global install of `web-component-tester` or `polyserve` (or both).
 
-1.  Clone the element repo.
+1. Clone the element repo.
 
-2.  Install the dependencies.
+1. Install the dependencies.
 
         bower install
 
-3.  Run the tests:
+1. Run the tests:
 
-	    wct
+       wct
 
-	Or run the tests in a browser:
+  Or run the tests in a browser:
```

Comment:
This needs 1 more indentation level:
![image](https://user-images.githubusercontent.com/5948271/34569093-fc70051e-f167-11e7-96e5-3e696ab81b39.png)
which was originally
![image](https://user-images.githubusercontent.com/5948271/34569104-038d1972-f168-11e7-8f8e-c04b8f8c7f72.png)